### PR TITLE
sign in sign out done

### PR DESCRIPTION
**Pull request by garfield414**

### CODE REVIEW

File: app/javascript/components/App.js [hunk 1]
```diff
@@ -12,18 +15,17 @@ class App extends Component {
...
+        <Router>
+          <Header {...this.props} />
+          <Switch>
+            {/* <Route exact path="/" component={Home} /> */}
+            {/* //<Route path="/catindex" component={CatIndex} /> */}
+            {/* //<Route path="/catshow" component={CatShow} /> */}
+            {/* <Route path="/users/sign_in" component={SignIn} /> */}
+            {/* //<Route path="/catedit" component={CatEdit} /> */}
+            {/* //</Switch></Router>Route component={NotFound} /> */}
+          </Switch>
```
Comment (sjproctor) [inline app/javascript/components/App.js:27]: You may want to remove the cat routes here. Definitely remove the `SignIn` route.  And looks like there are some extra tags on line 26.

File: app/javascript/components/components/Header.js [hunk 2]
```diff
@@ -28,32 +27,33 @@ export default class Header extends Component {
...
+      sign_out_route,
+    } = this.props;
     return (
       <div>
         <Navbar color="light" light expand="md">
           <NavbarBrand href="/">reactstrap</NavbarBrand>
           <NavbarToggler onClick={this.toggle} />
           <Collapse isOpen={this.state.isOpen} navbar>
             <Nav className="ml-auto" navbar>
+              {/* <NavItem>
```
Comment (sjproctor) [inline app/javascript/components/components/Header.js:44]: This can be removed.

File: app/javascript/components/components/SignIn.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import React, { Component } from "react";
```
Comment (sjproctor) [inline app/javascript/components/components/SignIn.js:1]: I don't think you will be using this component.

**Review by sjproctor — CHANGES_REQUESTED** [review]
Couple things to clean up.

File: app/javascript/components/App.js (hunk 1 above)
Comment (garfield414) [reply app/javascript/components/App.js:27]: Will do. I should have cleaned those things up yesterday as I was doing this. I will knock it out. Thank you.

**Review by sjproctor — APPROVED** [review]
Great!

**Pull request merged**